test/Core.Test/Vault/Services/CipherServiceTests.cs

```diff
@@ -625,7 +624,7 @@ public async Task SaveAsync_WrongRevisionDate_Throws(SutProvider<CipherService>
 
         await sutProvider.Sut.ShareManyAsync(cipherInfos, organization.Id, collectionIds, sharingUserId);
         await sutProvider.GetDependency<ICipherRepository>().Received(1).UpdateCiphersAsync(sharingUserId,
-            Arg.Is<IEnumerable<Cipher>>(arg => arg.Except(ciphers).IsNullOrEmpty()));
+            Arg.Is<IEnumerable<Cipher>>(arg => arg.Except(ciphers).Count() == 0));
```

Comment:
❓ Ugly? What's better?

Reply:
Is there a reason why this is preferred over the LINQ version? I think the LINQ version is clearer and more performant.

Reply:
`IsNullOrEmpty` no longer exists. Are you suggesting `Any()`?

Reply:
Yeah `!Any()`